Dockerfile

```diff
@@ -11,10 +13,10 @@ RUN ulimit -n 1024 && yum -y update && yum -y install \
     zip \
     && yum clean all
 
-RUN python3 -m pip install --upgrade pip \
-    # boto3 is available to lambda processes by default,
-    # but it's not in the amazonlinux image
-    && python3 -m pip install boto3
+RUN python3 -m pip install pip==18.1
+
+# Requirements copied form lambda Python 3.6, but not in base image
+RUN pip install -r quilt/requirements.txt
```

Comment:
Does it actually run the correct `pip`? In any case, change all lines to either run `pip` or `python3 -m pip`, for consistency.

Reply:
oh good catch